starter/script.js

```diff
@@ -1 +1,119 @@
-// put code here!
+
+var firstRound = function() {
+  console.log("It's your first day on the job as winter caretaker at the Overlook Hotel.  You and your family are looking forward to a nice relaxing winter.");
+  buffer();
+  console.log("So, What do you want to do first?\nWork on your novel?\nGo see what Danny is up to?\nOr ride your big wheel around the hotel?");
+  buffer();
+  console.log("Please enter '1' to work on your novel, '2' to go see Danny, or '3' to ride the big wheel.")
+  buffer();
+  var firstInput = prompt("Please enter either \n'1': to work on your novel,\n'2': to go see Danny,\n'3': to ride the big wheel.");
+
```

Comment:
reminder that prompt returns ever input as a string, so if you wanted to do a strict comparison in the checks below, `===`, you have to remember to parse the input into an int